state-recover/test-cases/src/integrationTest/kotlin/linea/staterecover/StateRecoveryManualReplayToLocalStackIntTest.kt

```diff
@@ -0,0 +1,134 @@
+import io.vertx.core.Vertx
+import io.vertx.junit5.VertxExtension
+import net.consensys.linea.BlockParameter
+import net.consensys.linea.jsonrpc.client.RequestRetryConfig
+import net.consensys.linea.jsonrpc.client.VertxHttpJsonRpcClientFactory
+import net.consensys.linea.testing.submission.AggregationAndBlobs
+import net.consensys.linea.testing.submission.loadBlobsAndAggregationsSortedAndGrouped
+import net.consensys.linea.testing.submission.submitBlobsAndAggregations
+import net.consensys.zkevm.ethereum.ContractsManager
+import net.consensys.zkevm.ethereum.LineaRollupDeploymentResult
+import net.consensys.zkevm.ethereum.Web3jClientManager
+import net.consensys.zkevm.ethereum.waitForTxReceipt
+import org.apache.logging.log4j.LogManager
+import org.assertj.core.api.Assertions.assertThat
+import org.awaitility.Awaitility.await
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.extension.ExtendWith
+import java.net.URI
+import kotlin.time.Duration.Companion.milliseconds
+import kotlin.time.Duration.Companion.minutes
+import kotlin.time.Duration.Companion.seconds
+import kotlin.time.toJavaDuration
+
+@ExtendWith(VertxExtension::class)
+class StateRecoveryManualReplayToLocalStackIntTest {
+  private val log = LogManager.getLogger("test.case.StateRecoverAppWithLocalStackIntTest")
+  private lateinit var aggregationsAndBlobs: List<AggregationAndBlobs>
+  private lateinit var stateManagerClient: StateManagerClientV1
+  private val testDataDir = "testdata/coordinator/prover/v3"
+
+  private val stateManagerUrl = "http://localhost:8890"
+
+  @BeforeEach
+  fun beforeEach(vertx: Vertx) {
+    aggregationsAndBlobs = loadBlobsAndAggregationsSortedAndGrouped(
+      blobsResponsesDir = "$testDataDir/compression/responses",
+      aggregationsResponsesDir = "$testDataDir/aggregation/responses"
+    )
+    val jsonRpcFactory = VertxHttpJsonRpcClientFactory(vertx = vertx, meterRegistry = SimpleMeterRegistry())
+
+    stateManagerClient = StateManagerV1JsonRpcClient.create(
+      rpcClientFactory = jsonRpcFactory,
+      endpoints = listOf(URI(stateManagerUrl)),
+      maxInflightRequestsPerClient = 1U,
+      requestRetry = RequestRetryConfig(
+        backoffDelay = 10.milliseconds,
+        timeout = 2.seconds
+      ),
+      zkStateManagerVersion = "2.3.0",
+      logger = LogManager.getLogger("test.clients.l1.state-manager")
+    )
+  }
+
+  private lateinit var rollupDeploymentResult: LineaRollupDeploymentResult
+
+  @Test
+  fun setupDeployContractForL2L1StateReplay() {
+    this.rollupDeploymentResult = ContractsManager.get()
+      .deployLineaRollup(numberOfOperators = 2, contractVersion = LineaContractVersion.V6).get()
+    log.info("""LineaRollup address=${rollupDeploymentResult.contractAddress}""")
+    log.info(
+      """
+      Start state recovery besu and shomei with the following configuration:
+       ./gradlew state-recover:besu-plugin:shadowJar \
```

Comment:
I think it would be better for the entrypoint to be just `make X` command instead